.github/workflows/run-tests.yml

```diff
@@ -29,5 +29,6 @@ jobs:
       #  run: |
       #    conda install pytest python-dateutil==2.8.2 pint==0.21
       - name: Test with pytest
+        working-directory: python/idsse_common/test
         run: |
-          pytest python/idsse_common/test --cov=python/idsse_common python/idsse_common/test --cov-report=term-missing
\ No newline at end of file
+          pytest --cov=../idsse/common --cov-report=term
```

Comment:
I didn't think the relative pathing would work in Github Actions, nice to see it does you got the coverage to be more accurate!